README.md

```diff
@@ -439,6 +439,31 @@ module.exports = {
 };
```

Comment:
Readme should also be updated not to mention that 0.8.1 is the default version (probably just removing that comment would be enough). I'm commenting here because I can't comment on the exact line.